### ISSUE TITLE
Could not load the ruby-graphiz or active_support gems for rendering: cannot load such file -- ruby-graphviz

### ISSUE THREAD

**Issue by stefanahman** [geekq/workflow-activerecord#4]
Hello,

I'm getting the following output when running the specs:

```
Could not load the ruby-graphiz or active_support gems for rendering: cannot load such file -- ruby-graphviz
```

This change (https://github.com/geekq/workflow-activerecord/commit/443fbc7eabe056b10cc6a10b42369d2f884888e9#diff-b73a1e5b99ecf4a35dfd425427c298ebR7) adds a warning every time `ruby-graphviz` cannot be loaded.

Related: https://github.com/geekq/workflow/issues/213

Thanks!

**Comment by geekq** [comment]
Just to let you know: I am on holiday till 4th of April

**Comment by wingice** [comment]
same here...

**Comment by geekq** [comment]
Should have been fixed since yesterday, commit https://github.com/geekq/workflow-activerecord/commit/fe8eb3b31314d833573557b6b4eebb8a07cddf5e.
Should work for you @stefanahman if you reference the `develop` branch:

    gem "workflow-activerecord", git: "git@github.com:geekq/workflow-activerecord.git", branch: "develop"

**Comment by stefanahman** [comment]
Works perfect! 👍

**Comment by stefanahman** [comment]
@geekq When will the new release be out?

**Comment by geekq** [comment]
workflow 2.0.1 released, workflow-activerecord 4.1.3 released !
